contracts/beacon/IVoucher.sol

```diff
@@ -32,6 +33,7 @@ interface IVoucher {
     function getVoucherHub() external view returns (address);
     function getType() external view returns (uint256);
     function getExpiration() external view returns (uint256);
+    function setExpiration(uint256 expiration) external;
```

Comment:
To put with non-view functions (same in implementation).
We can also decide to not follow the standard layout, we just need to agree on it.

Reply:
See [Move up setExpiration](https://github.com/iExecBlockchainComputing/iexec-voucher-contracts/pull/23/commits/010660fb95995c735e160cf607214326b31c4e16)